iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandler.java

```diff
@@ -270,6 +270,15 @@ public boolean supportsPartitionTransform() {
     return true;
   }
 
+  @Override
+  public Map<String, String> getPartitionTransformSpecProperty(org.apache.hadoop.hive.ql.metadata.Table hmsTable) {
+    Map<String, String> property = new HashMap<>();
+    TableDesc tableDesc = Utilities.getTableDesc(hmsTable);
+    Table table = Catalogs.loadTable(conf, tableDesc.getProperties());
```

Comment:
Would it make sense to use the `IcebergTableUtil` to load the table here? Not sure if there are other places too during `show create table` where we load the table, maybe there's some serde initialization at some point?

Reply:
The iceberg table is only loaded once, so changing the call to `IcebergTableUtil#getTable()` doesn't add much value at the moment. But I guess, we should consider that this might change in the future.